packages/maskbook/src/plugins/Wallet/UI/SelectProviderDialog.tsx

```diff
@@ -60,8 +75,12 @@ function SelectProviderDialogUI(props: SelectProviderDialogUIProps) {
     }, [setOpen])
     //#endregion
 
+    //#region select wallet dialog
+    const [, selectWalletDialogOpen] = useRemoteControlledDialog(WalletMessageCenter, 'selectWalletDialogUpdated')
```

Comment:
I'm afraid you will get a lot of merge conflict if you continuously working on released branch instead of master branch 👀

Reply:
Change base to master #1883.